docs/support.rst

```diff
@@ -0,0 +1,207 @@
+
+`Maya USD Documentation <https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2022/ENU/Maya-USD/files/GUID-9E9D45F2-4DA9-497B-8D69-1573ED6B2BA8-html.html>`_
+
+Multiverse for Maya
+^^^^^^^^^^^^^^^^^^^
+
+`Multiverse <https://j-cube.jp/solutions/multiverse/>`_ is another USD plugin for Maya that adds other pipeline features
+to Maya as well.
+
+Blender
+=======
+
+`Blender <https://www.blender.org>`_ is a free and open source 3D creation suite.
+
+`Blender USD Documentation <https://docs.blender.org/manual/en/latest/files/import_export/usd.html>`_
+
+
+Foundry
+=======
+
+The Foundry are a developer of 3D Content Creation, Lighting and Compositing software.
+
+Katana
+------
+
+`Katana <https://www.foundry.com/products/katana>`_ is  look development and lighting package.
+
+Katana has `open source USD plugins <https://learn.foundry.com/katana/dev-guide/Plugins/KatanaUSDPlugins/index.html>`_ as
+well as a support for `Hydra render delegates <https://learn.foundry.com/katana/dev-guide/Plugins/HydraRenderDelegates/index.html>`_.
+
+Modo
+----
+
+`Modo <https://www.foundry.com/products/modo>`_ is a general purpose 3D package.
+
+Nuke
+----
+
+`Nuke <https://www.foundry.com/products/nuke-family/nuke>`_ is a compositing package with support for both USD and Hydra.
+
+`Nuke documentation <https://learn.foundry.com/nuke/content/comp_environment/3d_compositing/usd.html>`_
+
+ZBrush
+------
+
+`ZBrush <https://www.maxon.net/en/zbrush>`_ is a 3D sculpting and painting application.
+
+`ZBrush USD documenation <http://docs.pixologic.com/user-guide/zbrush-other-programs/usd-format/>`_
+
+Houdini
+=======
+
+`Houdini <https://www.sidefx.com/products/houdini/>`_ is a 3D package with a focus on procedural content and effects.
+
+Houdini includes `Solar <https://www.sidefx.com/products/houdini/solaris/>`_ to create composed USD content.
+
+`SideFX USD Documentation <https://www.sidefx.com/docs/houdini/solaris/usd.html>`_
+
+
+Maxon
+=====
+
+Maxon are a developer of 3D content creation, rendering and motion graphics software.
+
+Cinema 4D
+---------
+
+`Cinema 4D <https://www.maxon.net/en/cinema-4d>`_ is a general purpose 3D package.
+
+Redshift
+--------
+
+`Redshift <https://www.maxon.net/en/redshift>`_ is a GPU-accelerated path-traced renderer.
+
+Redshift supports USD and is also available as a Hydra delegate for rendering integration.
+
+
+Omniverse
+=========
+
+`NVIDIA Omniverse <https://www.nvidia.com/en-us/omniverse/>`_ is a platform for creating and operating metaverse applications.
+It is based on USD.
+
+`NVIDIA USD Documentation <https://www.nvidia.com/en-us/omniverse/usd/>`_
+
+Unity
+=====
+
+`Unity <https://unity.com>`_ is a real time 3D engine and editor.
+It includes an `open source USD Package <https://github.com/Unity-Technologies/usd-unity-sdk>`_.
+
+`Unity USD Documentation <https://docs.unity3d.com/2020.1/Documentation/Manual/com.unity.formats.usd.html>`_
```

Comment:
I think this is a better/more direct link to the docs for the most recent package release:
https://docs.unity3d.com/Packages/com.unity.formats.usd@3.0/manual/index.html